libraries/apollo-normalized-cache-incubating/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/ClientCacheExtensions.kt

```diff
@@ -44,8 +42,9 @@ enum class FetchPolicy {
   /**
    * Try the cache, if that failed, try the network.
    *
-   * An [ApolloCompositeException] is thrown if the data is not in the cache and the network call failed.
-   * If coming from the cache 1 value is emitted, otherwise 1 or multiple values can be emitted from the network.
+   * 1 value from the cache is emitted first (with a non null [ApolloResponse.exception] if it's a cache miss),
+   * and then, if it's a cache miss, 1 or multiple values from the network are emitted (with a non null [ApolloResponse.exception] if there
+   * was a network error).
```

Comment:
```suggestion
   * This [FetchPolicy] emits one or several [ApolloResponse]s. 
   * Cache misses and network errors have [ApolloResponse.exception] set to a non-null [ApolloException]
```

Reply:
Tried simplifying and uniformizing the wording. Let me know what you think

Reply:
Yes, this is better! 👍

Reply:
I pushed a change to use the same wording in the non-incubating normalized cache (and also remove a non-breaking space that made it through the GitHub UI)